src/components/FocusTrap/SCREENS_WITH_AUTOFOCUS.ts

```diff
@@ -0,0 +1,16 @@
+import {CENTRAL_PANE_WORKSPACE_SCREENS} from '@libs/Navigation/AppNavigator/Navigators/FullScreenNavigator';
+import SCREENS from '@src/SCREENS';
+
+const SCREENS_WITH_AUTOFOCUS: string[] = [
```

Comment:
This seems pretty manual/brittle. Is there a way we can derive which screens have auto-focus or not, rather than having a const we need to remember to update?

Reply:
I don't think there is any other way to configure this. Maybe only by rewriting configuration in `src/SCREENS.ts` and enforcing this information there.
cc @adamgrzybowski

Reply:
If I remember correctly I tried two different approaches:
1. Check if the input is focused -> That caused many problems because autofocusing is often applied with a little delay. e.g. after the screen stops animating. I couldn't get a satisfying solution this way

2. There is a hook useAutofocusInput. I thought that we could check the name of the screen here and add it to the array. After some investigation (correct me if I am wrong) it turned out that it's not the only way we handle autofocus. In some places, it's just hardcoded without some general hook.

When I created this PR, the array with const seemed as the simplest and most reliable solution. 

If we unify how we handle autofocus, we could once again consider option nr. 2. Or maybe there is another option I haven't consider